--- origsrc/libusb-compat-0.1.3/libusb/Makefile.am	2008-06-17 18:13:02.000000000 -0500
+++ src/libusb-compat-0.1.3/libusb/Makefile.am	2010-06-01 03:24:18.540886400 -0500
@@ -5,5 +5,5 @@ libusb_la_SOURCES = core.c usbi.h
 libusb_la_CFLAGS = -fvisibility=hidden $(AM_CFLAGS) $(LIBUSB_1_0_CFLAGS)
 libusb_la_LIBADD = $(LIBUSB_1_0_LIBS)
 libusb_la_LDFLAGS = -version-info $(LT_MAJOR):$(LT_REVISION):$(LT_AGE) \
-	-release 0.1
+	-release 0.1 -no-undefined
 
